include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -3219,17 +3285,17 @@ __parallel_set_union_op(_ExecutionPolicy&& __exec, _ForwardIterator1 __first1, _
                                       __copy_range, ::std::true_type());
             },
             [=, &__result] {
-                __result = __parallel_set_op(::std::forward<_ExecutionPolicy>(__exec), __first1, __last1,
-                                             __left_bound_seq_2, __last2, __result, __comp,
-                                             [](_DifferenceType __n, _DifferenceType __m) { return __n + __m; },
-                                             __set_union_op, __is_vector);
+                __result = __parallel_set_op(
+                    ::std::forward<_ExecutionPolicy>(__exec), __first1, __last1, __left_bound_seq_2, __last2, __result,
+                    __comp, [](_DifferenceType __n, _DifferenceType __m) { return __n + __m; }, __set_union_op,
+                    __is_vector);
             });
         return __result;
     }
 
-    return __parallel_set_op(::std::forward<_ExecutionPolicy>(__exec), __first1, __last1, __first2, __last2, __result,
-                             __comp, [](_DifferenceType __n, _DifferenceType __m) { return __n + __m; }, __set_union_op,
-                             __is_vector);
+    return __parallel_set_op(
+        ::std::forward<_ExecutionPolicy>(__exec), __first1, __last1, __first2, __last2, __result, __comp,
```

Comment:
General question - why are there so many format style changes? It look like that you have the other options for `clang format` - f.e. 100 symbols per line instead of 120. See https://github.com/oneapi-src/oneDPL/blob/main/.clang-format